framework/src/base/MooseApp.C

```diff
@@ -129,6 +130,13 @@ MooseApp::validParams()
       false,
       "Ignore input file and build a minimal application with Transient executioner.");
 
+#ifdef WASP_ENABLED
+  params.addCommandLineParam<bool>(
+      "language_server",
+      "--language-server",
+      "Starts a process to communicate with development tools using the language server protocol");
+#endif
```

Comment:
Our Workbench integration plan is to grep the `app-opt --help` output to see if the language server is available.

 - if `--language-server` is in the available options, then we will start and use the MOOSE language server
 
 - if it doesn't show up in the options, we will fall back to the old `--definition` schema based integration

Otherwise, Workbench will have to blindly try starting the language server then wait a millisecond for it to error.

After which that subprocess would be killed and Workbench would fall back to the `--definition` schema route.

We would prefer to efficiently check which integration route to take from the options in `app-opt --help` output.

But if there's a strong preference to always show the option even when it's not available, I can make this change.

Reply:
This is fine.